### PR TITLE
dtoverlay: add dtoverlay tag with legacy rpi-fbcp

### PR DESCRIPTION
**Pull request by klutchell**
Allow a choice between fbcp-ili9341 and rpi-fbcp by adding a new tag
for dtoverlay. This tag will rely on the host OS device tree overlays and
the user setting the BALENA_HOST_CONFIG_dtoverlay device config environment variable.

This should be considered a legacy option, but likely has support
for more screen types than fbcp-ili9341 alone. This tag also includes
the dbus workaround to disable plymouth.

Change-type: patch
Signed-off-by: Kyle Harding <kyle@balena.io>

### CODE REVIEW

**Comment by rahul-thakoor** [comment]
hey @klutchell what is the final image size?

I would like to try using the buildroot approach to slim down the images. thoughts?

**Comment by klutchell** [comment]
Started to rough in support for buildroot, but the the CMakeLists need to be patched still.

```
export DOCKER_BUILDKIT=1
docker build buildroot
```

File: buildroot/package/rpi-fbcp/rpi-fbcp.mk [hunk 1]
```diff
@@ -0,0 +1,12 @@
+################################################################################
```
Comment (jakogut) [inline buildroot/package/rpi-fbcp/rpi-fbcp.mk]: Nice job on this package, is this patch submitted upstream?

I searched the buildroot mailing list, and in the process, found that somebody else submitted an rpi-fbcp package. A reviewer left feedback on it, but it was abandoned.

Comment (klutchell) [reply buildroot/package/rpi-fbcp/rpi-fbcp.mk:1]: It's not submitted upstream as I haven't been able to get it to test it yet! So far all my builds seem to just skip this recipe even though it's been added to packages and included in my config. Can you see what I'm doing wrong?

**Comment by klutchell** [comment]
https://github.com/juj/fbcp-ili9341/issues/50

**Pull request closed without merging**